lib/bolt/inventory/group.rb

```diff
@@ -52,13 +52,7 @@ def initialize(data)
             raise ValidationError.new("Node entry must be a String or Hash, not #{node.class}", @name)
           end
 
-          if @nodes.include?(node['name'])
```

Comment:
I'm not sure I understand why inventory and group v1 need to change? I think the bug in the ticket only occurs with inventory v2.